tests/test_iam.py

```diff
@@ -1055,6 +1055,68 @@ def test_iam_group_delete(test, iam_user_group):
         client.get_group(GroupName=resources[0]['GroupName'])
 
 
+@terraform('iam_delete_certificate', teardown=terraform.TEARDOWN_IGNORE)
+def test_iam_delete_certificate_action(test, iam_delete_certificate):
+    iam_name = iam_delete_certificate[
+        'aws_iam_server_certificate.test_cert_alt.name']
+
+    session_factory = test.replay_flight_data('iam_delete_certificate')
+    client = session_factory().client('iam')
+
+    try:
+        client.get_server_certificate(ServerCertificateName=iam_name)
```

Comment:
remove this api call, and add the assertion on server cert name to the policy as an attribute filter